client-v2/src/components/FrontsEdit/ArticleFragmentForm.tsx

```diff
@@ -357,18 +359,45 @@ class FormComponent extends React.Component<Props, FormComponentState> {
           <RowContainer>
             <Row>
               <Col>
-                <ImageWrapper faded={imageHide}>
-                  <ConditionalField
-                    permittedFields={editableFields}
-                    name="primaryImage"
-                    component={InputImage}
-                    disabled={imageHide}
-                    criteria={imageCriteria}
-                    frontId={frontId}
+                {imageReplace && (
+                  <ImageWrapper faded={imageHide}>
+                    <ConditionalField
+                      permittedFields={editableFields}
+                      name="primaryImage"
+                      component={InputImage}
+                      disabled={imageHide}
+                      criteria={imageCriteria}
+                      frontId={frontId}
+                    />
+                  </ImageWrapper>
+                )}
+                {!imageReplace && (
+                  <ThumbnailEditForm
```

Comment:
Just to check -- 

- what happens here when there's no image for the article? (not common, but I think possible?)
- I presume the difference between this and an image that the user had added in fronts is the presence of a rubbish bin?

Reply:
- Nothing particularly interesting happens if the image is missing, we just see a blank where an image should be.
- Yes, if the user has added the image there is a rubbish bin and the imageReplace toggle is turned on.

Reply:
👍